microprofile.jdt/org.eclipse.lsp4mp.jdt.test/projects/maven/microprofile-configproperties/src/main/java/org/acme/GreetingResource.java

```diff
@@ -0,0 +1,16 @@
+package org.acme;
```

Comment:
Remove this file if you don't test against it

Reply:
fixed